packages/rspack/src/utils/with-react.ts

```diff
@@ -1,77 +1,29 @@
 import { Configuration } from '@rspack/core';
-import { withWeb } from './with-web';
 import { NxRspackExecutionContext } from './config';
+import { withWeb, WithWebOptions } from './with-web';
+import { applyReactConfig } from './lib/apply-react-config';
 
-export function withReact(opts = {}) {
+export interface SvgrOptions {
```

Comment:
You'll need to move this out of here and likely into `src/plugins/utils/models.ts`

There's a circular dep now between `with-react` and `apply-react-config`